app/src/test/java/io/gnosis/safe/ui/safe/terms/TermsCheckerTest.kt

```diff
@@ -0,0 +1,55 @@
+package io.gnosis.safe.ui.safe.terms
+
+import android.app.Application
+import io.gnosis.safe.ui.safe.terms.TermsChecker.Companion.TERMS_AGREED
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.spyk
+import junit.framework.Assert.assertFalse
+import junit.framework.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import pm.gnosis.svalinn.common.PreferencesManager
+import pm.gnosis.tests.utils.TestPreferences
+
+class TermsCheckerTest {
+    private lateinit var preferences: TestPreferences
+    private lateinit var preferencesManager: PreferencesManager
+
+    @Before
+    fun setup() {
+        preferences = spyk()
+        val application = mockk<Application>().apply {
+            every { getSharedPreferences(any(), any()) } returns preferences
+        }
+        preferencesManager = PreferencesManager(application)
+    }
+
+    @Test
+    fun `When terms agreed preferences should contain true`() {
```

Comment:
This is something we should have probably communicated to you sooner, but the convention for naming tests is:
For unit tests:
`methodName (input and precondition brief description) should <succeed|fail|returns|throws (type|value)>`
For instrumented tests we use underscores instead of whitespace and double underscore to replace parenthesis for the input section. If it's unclear you can see other test or let's have a chat.

Reply:
Done in 14e215271c75c8b1d0a0a002d38a06bd6af89c49